fiboa_cli/__init__.py

```diff
@@ -343,6 +343,7 @@ def jsonschema(schema, out, fiboa_version, id_):
 ## CONVERT
 @click.command()
 @click.argument('dataset', nargs=1, type=click.Choice(list_all_converter_ids()))
+@click.argument('variant', required=False)
```

Comment:
What's variant doing? It should have som documentation.
I think if it's optional I'd prefer it being an option instead of an argument. If it's for choosing a year, should we just call it time/year/... or so?

Reply:
It's for year. I just thought that **year** might be too specific. E.g. if there's a Converter that handles different regions, or different languages, or something else...

I will change it to an option `--year`...

Reply:
Resolved with https://github.com/fiboa/cli/pull/138/commits/249715ec4dbcae5df844ed9d44dd1eafaf5ccb69